### ISSUE TITLE
Show dependencies

### ISSUE THREAD

**Issue by wilzbach** [biojs/registry-ui#11]
The BioJS registry could show used npm dependencies and/or other dependencies (e.g. css) specified in the sniper section.

**Comment by emepyc** [comment]
Yes, I'm planning to look into this as an exercise to work with the registry
